src/models/create_token_transaction.ts

```diff
@@ -242,6 +251,71 @@ class CreateTokenTransaction extends Transaction {
 
     return tx;
   }
+
+  /**
+   * Checks if this transaction is the creation of an NFT following the NFT Standard Creation.
+   * @see https://github.com/HathorNetwork/rfcs/blob/master/text/0032-nft-standard.md#transaction-standard
+   * @throws {NftValidationError} Will throw an error if the NFT is not valid
+   *
+   * @param {Network} network Network to get output addresses first byte
+   * @returns {void} If this function does not throw, the NFT is valid
+   */
+  validateNftCreation(network: Network): void {
```

Comment:
Maybe change the name to `validateNFT` since this is a `create_token_transaction` it will, by default, be a creation

Reply:
Makes sense, refactored on [36d545f9](https://github.com/HathorNetwork/hathor-wallet-lib/pull/378/commits/36d545f9d9a288905c37000e85b78794faeda026).